### PR TITLE
Order Product Cart

### PR DESCRIPTION
**Pull request by Rajkumar-Selvaraj**
I have added OrderProduct entity and also created one Api which is used to place the order.

### CODE REVIEW

File: src/main/resources/application.properties
Comment (jithuzuci) [inline src/main/resources/application.properties:1]: I don't think you need to add properties file changes

**Review by jithuzuci — COMMENTED** [review]
Please minimize the usage of commented code while trying to merge the PR to the `master` branch. It should always be in good structure and good enough for anyone to check out and use. 

Please add test cases for the logics which you have added

File: src/main/resources/application.properties
Comment (Rajkumar-Selvaraj) [reply src/main/resources/application.properties:1]: I have resolved the changes. Actually, The password for my Db user is different from Mariyappan's Password. I forgot to change it when I was creating the Pull request.

**Review by jithuzuci — APPROVED** [review]
Code overall looks good, I appreciate it if you could make smaller PRs in the future

**Comment by Rajkumar-Selvaraj** [comment]
Ok Jithu, I will do small PR in the future.

**Pull request merged**